tests/VolleyManagement.UnitTests/Services/TeamService/TeamServiceTests.cs

```diff
@@ -160,25 +162,22 @@ public void GetTeamCaptain_CaptainExist_PlayerReturned()
         /// Test for Create() method. The method should create a new team.
         /// </summary>
         [TestMethod]
-        public void Create_TeamPassed_TeamCreated()
+        public void Create_CreateTeamDtoPassed_DtoAddedByRepository()
```

Comment:
Why did you rename last part?

Reply:
To follow the 'Assert' part of test.
Is it better to revert change of the name, along with adding some logic so it would follow the description?

Reply:
In Assert part of the test you have "VerifyCreateTeam". I do not follow you probably.